cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs

```diff
@@ -306,10 +307,10 @@ runGenesisCreate (GenesisDir rootdir)
   let finalGenesis = updateTemplate start mAmount genDlgs utxoAddrs template
 
   writeShelleyGenesis (rootdir </> "genesis.json") finalGenesis
-  where
-    gendir  = rootdir </> "genesis-keys"
-    deldir  = rootdir </> "delegate-keys"
-    utxodir = rootdir </> "utxo-keys"
+  where adjustTemplate t = t { sgNetworkMagic = unNetworkMagic (toNetworkMagic network) }
+        gendir  = rootdir </> "genesis-keys"
+        deldir  = rootdir </> "delegate-keys"
+        utxodir = rootdir </> "utxo-keys"
```

Comment:
Please match the formating you are replacing.

Reply:
Fixed!